k9mail/src/main/java/com/fsck/k9/preferences/AccountSettings.java

```diff
@@ -365,16 +372,18 @@ public Object fromString(String value) {
 
     /**
      * The delete policy setting.
+     *
+     * Formerly an int, converted to an enum.
```

Comment:
I added the comment to explain why, unlike the other enum preferences, this is stored as an int. Shall I update the comment to make that clear, or just delete it?